hack/generated/pkg/testcommon/test_context.go

```diff
@@ -210,3 +213,13 @@ func (tc PerTestContext) NewTestResourceGroup() *resources.ResourceGroup {
 		},
 	}
 }
+
+func (tc PerTestContext) MakeARMId(resourceGroup string, provider string, params ...string) string {
+	if len(params)%2 != 0 {
+		panic("ARM Id params must come in resourceKind/name pairs")
+	}
+
+	suffix := strings.Join(params, "/")
+
+	return fmt.Sprintf("/subscriptions/%s/resourceGroups/%s/providers/%s/%s", tc.AzureSubscription, resourceGroup, provider, suffix)
```

Comment:
This will have a trailing `/` if `params` is empty - is that correct?

Reply:
No, but really `params` min length should be 2. I'll add a check for that.